File: cordova-plugin-ownid/src/ios/OwnIDCordovaPlugin.swift
import Foundation
import WebKit
import OwnIDCoreSDK

@objc(OwnIDCordovaPlugin) class OwnIDCordovaPlugin: CDVPlugin {
    let webBridge = OwnID.CoreSDK.createWebViewBridge()
    
    override func pluginInitialize() {
        super.pluginInitialize()

        OwnID.CoreSDK.configure(userFacingSDK: Self.info())

        if let webView = webView as? WKWebView {
            webBridge.injectInto(webView: webView)
        }
    }

    private static func info() -> OwnID.CoreSDK.SDKInformation {
        ("OwnIDCordovaPlugin", "3.5.0")
    }
}

File: cordova-plugin-ownid/src/android/com/ownid/sdk/cordova/OwnIdCordovaPlugin.java
package com.ownid.sdk.cordova;

import android.webkit.WebView;

import com.ownid.sdk.OwnId;
import com.ownid.sdk.OwnIdWebViewBridge;

import org.apache.cordova.CordovaPlugin;

import java.util.Collections;
import java.util.HashSet;

public class OwnIdCordovaPlugin extends CordovaPlugin {

    @Override
    protected void pluginInitialize() {
        super.pluginInitialize();

        OwnId.createInstanceFromFile(cordova.getActivity(), "ownIdSdkConfig.json", "OwnIdCordovaPlugin/3.4.0");

        OwnIdWebViewBridge webViewBridge = OwnId.createWebViewBridge();
        HashSet<String> allowedOriginRules = new HashSet<>(Collections.singletonList("https://localhost"));
        webViewBridge.injectInto((WebView) webView.getView(), allowedOriginRules, cordova.getActivity(), true, null);
    }
}